### ISSUE TITLE
Ctrl/Alt prefixed shortcuts aren't passed properly

### ISSUE THREAD

**Issue by somasis** [91861/wayst#26]
I *think* this has to do with the to-do item, "all xterm and vte control sequences", but I'm not sure so I figure I should report it anyway.

Keyboard shortcuts like alt+tab, alt+A, alt+backspace, and same with ctrl+*, don't seem to be properly passed through to the application running in the terminal.

**Comment by somasis** [comment]
By the way, this terminal is really impressive so far. I'm definitely going to keep my eye on this project, as I'm not really satisfied with my current emulator (alacritty) because it just doesn't seem very lightweight. Thank you for your work so far!

**Comment by shoober420** [comment]
@somasis 
I agree, alacritty seems bloated for sure. I used to use “st” as my main, but always wanted GPU acceleration. Wayst is perfect. It has the lightweight feeling of “st”, with the power of alacritty. I’ve been using wayst as my main for about two weeks now and I love it. @91861 and @freed00m are doing a fantastic job. I love your terminal emulator.

**Comment by freed00m** [comment]
@shoober420 - I don't deserve the credit :) I just created the AUR package to raise awareness about this awesome terminal.

Hopefully I will have time soon to help with code as well.

**Comment by shoober420** [comment]
I felt I should mention that when attempting to use the vim yank method for copying and pasting, instead of the desired text being yanked, pressing Ctrl+Shift+P will paste the last command entered in the terminal.
